Assets/Scripts/Explosion.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Explosion : MonoBehaviour {
+
+	[SerializeField]
+	private float explosionSize;
```

Comment:
this needs to be clearer, such as maxExplosionRadius

Reply:
You need to do some assert checks and `[Range(min,max)]` to insure that the developer/designer doesn't make a mistake this